screens/SignUp.js

```diff
@@ -14,8 +14,8 @@ const { height, width } = Dimensions.get("screen");
 import { useNavigation } from "@react-navigation/native";
```

Comment:
In Firestore documents were getting stored to a newly generated doc ref id but it should have been with the userUid which was generated when the user passed the onboarding pages so used a different function to do that